mtags/src/main/scala/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -39,44 +39,49 @@ class CompletionProvider(
       cursor = Some(params.offset),
       cursorName = cursorName
     )
+
     val pos = unit.position(params.offset)
     val isSnippet = isSnippetEnabled(pos, params.text())
-    val clientSupportsSnippets =
```

Comment:
I believe this is needed a bit further down to get the correct suffixes?

Reply:
We have access to it already from a val in a trait that's mixed in here (`Completions#clientSupportsSnippets`). I believe it's the same value...

Reply:
Ahh, I didn't notice that. Cool!